creator/views.py

```diff
@@ -0,0 +1,31 @@
+from graphene_file_upload.django import FileUploadGraphQLView
+
+
+class SentryGraphQLView(FileUploadGraphQLView):
+    """
+    Sets the Sentry transaction name to be that of the GraphQL operation.
+    This helps break down all the /graphql requests into more actionable
+    buckets within Sentry.
+    """
+
+    def execute_graphql_request(self, *args, **kwargs):
+        """
+        Attempt to extract the operation name and use it to set the Sentry
```

Comment:
Curious, before this change what's an example transaction name and after this what will the transaction names look like?